### DISCUSSION TITLE
Welcome to bdsim Discussions!

### DISCUSSION THREAD

**Discussion by petercorke** [petercorke/bdsim#12] [General]
## Welcome!

I've set up Discussions as a place to connect with other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by Ahmed8395** [comment]
Hello Peter,
I really cannot thank you enough for the work you have done as I have learnt a great deal about robotics following your content. I recently bought your book and I have been following it and I am currently studying chapter 4 which is about mobile robots. I have ran into a problem when trying to launch **bdedit**, every time I run the file (bdedit.py) I get an error message saying (**No module named 'PyQt5'**) 
I would really appreciate the help. Thanks.

**Reply by petercorke** [reply]
Could you please post this as an issue.  Be sure to include your operating system.  Note that `bdedit.py` only provides a way to view/edit diagrams --  it is not necessary to execute them.
